src/lib/gist_backup.sh.in

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# gist_backup.sh : Enables backing up package list on github gist
+# This file is part of Yaourt (http://archlinux.fr/yaourt-en)
+
+gist_backup(){
+    # Get yaourtrc
+    [[ -r '@sysconfdir@/yaourtrc' ]] && source '@sysconfdir@/yaourtrc'
+    XDG_YAOURT_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/yaourt"
+    if [[ -r $XDG_YAOURT_DIR/yaourtrc ]]; then
+      source "$XDG_YAOURT_DIR/yaourtrc"
+    elif [[ -r ~/.yaourtrc ]]; then
+      source ~/.yaourtrc
+    fi
+
+    # Determine if Fresh Install is Needed
+    if [ -z "$GIST_ID" ] || [ $FORCE -eq 1 ]; then
+      gist_backup_install
+    else
+      gist_backup_update;
+    fi
+    exit 0;
+}
+
+gist_backup_poll(){
+  # Get yaourtrc
+  [[ -r '/usr/etc/yaourtrc' ]] && source '/usr/etc/yaourtrc'
```

Comment:
Initially I was using the /etc/yaourtrc file, but all of the documentation I have found seems to suggest that the ~/.yaourtrc file is better sandboxed from other users, which would make this more reliable on multi-tenant environments.